### ISSUE TITLE
Generate random decks

### ISSUE THREAD

**Issue by nwawrzyniak** [feildmaster/UnderScript#32]
Possibility to create a random deck from **your** collection.

This would mean having to parse/know the players entire collection. Normal cards as well as the shiny amounts.
The decks could be either gold-curve-balanced with some cards of every cost or maybe fetch like 10 randomly cards selected cards and fill the deck with cards that share synergies like keywords and tribes. I Don't really care about how the random deck works tbh as long as decks are somewhat playable while completely random and not only the highest cost cards or something.

For comparison: Hearthstone got a similar feature and goes so far to take card synergy statistics into account and fills decks after a few random cards with the cards that are most oftenly played along them. A little overkill but creates cool, smart, random decks.

**Comment by feildmaster** [comment]
I'll be honest, "somewhat playable" is pretty impossible without making underscript have 5000 lines of code just to generate the deck

**Comment by nwawrzyniak** [comment]
Nah, the deck should already be (somewhat) playable once you just add cards that share a single keyword, for example creating full taunt decks by adding a lot of cards that include the word "taunt", a silencer deck by adding cards with the keyword "silence" and so on. These decks may also contain anti-synergy (e.g. if there was a spell that deals less damage the more taunt monsters are on board it would still be included in the deck), but this would be a way to reduce the amount of lines needed to well under 100.

TL;DR: You're thinking to complicated. Deck generation does not need to be that smart.

**Comment by feildmaster** [comment]
Ah, completely skipped over that part... Right now I'm just thinking of filling it completely randomly... I'll think of more logic later.

So long as I get a good base I can plan other things later, I suppose.
